src/main/resources/default-lintConfig.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is up to date with the current version of config file in that PR. It's gonna be a while before @mpeng3 have time to respond to those comments and make changes. Should I wait for that before merging this PR or merge this PR now and create then v1.0.1 with the lint changes later?